src/sections/Projects/Sistent/components/list/index.js

```diff
@@ -0,0 +1,210 @@
+  const location = useLocation();
+  const { isDark } = useStyledDarkMode();
+
+  return (
+    <SistentLayout title="List">
+      <div className="content">
+        <a id="Identity">
+          <h2>List</h2>
+        </a>
+        <p>
+          Lists are essential UI elements that allow items to be organized sequentially in a structured and readable way. They help users view, select, and interact with multiple items conveniently.
+        </p>
+        <div className="filterBtns">
+          <TabButton
+            className={location.pathname === "/projects/sistent/components/list" ? "active" : ""}
+            onClick={() => navigate("/projects/sistent/components/list")}
+            title="Overview"
+          />
+          <TabButton
+            className={location.pathname === "/projects/sistent/components/list/guidance" ? "active" : ""}
+            onClick={() => navigate("/projects/sistent/components/list/guidance")}
+            title="Guidance"
+          />
+          <TabButton
+            className={location.pathname === "/projects/sistent/components/list/code" ? "active" : ""}
+            onClick={() => navigate("/projects/sistent/components/list/code")}
+            title="Code"
+          />
+        </div>
+        <div className="main-content">
+          <a id="List Components">
+          </a>
+          <p>The List component is a flexible and structured container for organizing content in vertical layouts. It supports various child components, such as List Items, Icons, Avatars, Buttons, and Subheaders, making it ideal for creating navigational menus, data displays, or interactive content groups.</p>
+          <p>
+            The List component is designed to:
+          </p>
+          <ul>
+            <li><strong>Organize Content:</strong> Provides a structured vertical layout for related items.</li>
+            <li><strong>Interactive Elements:</strong> Add actionable buttons and components to enhance user interaction.</li>
+            <li><strong>Customizable Design:</strong> Offers size variations, dense modes, and alignment options.</li>
+            <li><strong>Accessibility:</strong> Ensures keyboard navigation and visual feedback for enhanced usability.</li>
+          </ul>
+          <a id="Components Overview">
+          </a>
+          <p>
+            Components Overview:
+          </p>
+
+          <ul>
+            <li>List Item: Represents an individual item within the list.</li>
+            <li>List Item Button: A clickable button within a list item.</li>
+            <li>List Item Icon: Displays an icon associated with a list item.</li>
+            <li>List Item Avatar: Adds an avatar for visual representation within the list item.</li>
+            <li>List Item Text: Contains the primary and secondary text within a list item.</li>
+            <li>List Subheader: Provides a labeled header for grouping related list items.</li>
+
+          </ul>
+          <a id="Types of List component">
```

Comment:
How a link should be added to the sidebar
```suggestion
          <a id="Types of List component">
```
```suggestion
          <a id="Types of List component">
            <h3> Types of List component </h3>
          </a>
```

Reply:
ok, sir.